programs/survival/waypoints.sc

```diff
@@ -7,7 +7,7 @@ global_waypoint_config = {
     // 4 : ALWAYS
     'allow_tp' -> 2
 };
-
+print(player());
```

Comment:
Assuming this is debug code that shouldn't be there?

Reply:
Indeed. I totally missed it because, since it's a global app, the essage is only printe on server startup, so I never saw it printing.